integration-tests/src/tests/transaction_payment.rs

```diff
@@ -1,4 +1,181 @@
+use crate::{
+	constants, constants::PricesBuilder, polimec, PolimecAccountId, PolimecBalances, PolimecCall, PolimecForeignAssets,
+	PolimecNet, PolimecOrigin, PolimecRuntime, PolimecSystem,
+};
+use frame_support::{
+	dispatch::GetDispatchInfo,
+	traits::{fungible::Mutate as FungibleMutate, fungibles, fungibles::Mutate as FungiblesMutate},
+};
+use macros::generate_accounts;
+use pallet_funding::{AcceptedFundingAsset, PriceProviderOf};
+use pallet_transaction_payment::FeeDetails;
+use parity_scale_codec::Encode;
+use polimec_common::{ProvideAssetPrice, PLMC_DECIMALS, PLMC_FOREIGN_ID, USD_DECIMALS};
+use polimec_runtime::Header;
+use sp_arithmetic::{FixedPointNumber, FixedU128};
+use sp_core::H256;
+use sp_runtime::{
+	traits::{Dispatchable, Header as HeaderT, SignedExtension},
+	DigestItem,
+};
+use xcm_emulator::TestExt;
+generate_accounts!(ALICE, AUTHOR);
+use frame_support::traits::fungible::Inspect;
```

Comment:
```suggestion
use frame_support::traits::fungible::Inspect;

generate_accounts!(ALICE, AUTHOR);
```

plus `cargo +nightly fmt --alll`

Reply:
doing fmt changes might fuck up the PRs on top. Let's make sure teh fmt check passes on the final PR before merging the stack